src/MixerBoard.cpp

```diff
@@ -601,63 +627,53 @@ void MixerTrackCluster::UpdateMeter(const double t0, const double t1)
    // of the meter display, but instead consult PlaybackPolicy
 
    const auto pTrack = GetWave();
-   auto startSample = (sampleCount)((pTrack->GetRate() * t0) + 0.5);
-   auto scnFrames = (sampleCount)((pTrack->GetRate() * (t1 - t0)) + 0.5);
+
+   // Don't throw on read error in this drawing update routine
+   constexpr auto mayThrow = false;
+   mSampleView = pTrack->GetSampleView(t0, t1, mayThrow);
```

Comment:
Something very basic:  Does `mSampleView` really need a longer lifetime than this function?  I think not, it can be a local, not a member.

Reply:
It does, if we want the caching to work properly.
Actually I notice one improvement: emptying this view upon start/stop so as to allow cache eviction.